packages/app/src/firebaseNamespaceCore.ts

```diff
@@ -222,11 +222,15 @@ export function createFirebaseNamespaceCore(
     namespace[name] = serviceNamespace;
 
     // Patch the FirebaseAppImpl prototype
+    // TODO: The eslint disable can be removed and the 'ignoreRestArgs' option added to
```

Comment:
nit: move it to directly above the eslint suppression

Reply:
Done.